Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -133,39 +133,49 @@ public void clear() {
     }
 
     void deleteSelectedStructure() {
-        Optional<IncludedStructuralElement> selectedStructure = getSelectedStructure();
-        if (!selectedStructure.isPresent()) {
+        if (!getSelectedStructure().isPresent()) {
             /*
              * No element is selected or the selected element is not a structure
              * but, for example, a media unit.
              */
             return;
         }
-        LinkedList<IncludedStructuralElement> ancestors = MetadataEditor.getAncestorsOfStructure(selectedStructure.get(), structure);
+        IncludedStructuralElement selectedStructure = getSelectedStructure().get();
```

Comment:
Please don’t rely on `get()`. The method returns a conditional for a reason. Please add an error message:

    getSelectedStructure().orElseThrow(() -> new IllegalStateException("No structure selected"))

Reply:
There is already a check whether the structure element is present in line 136 and the method returns if that is not the case, so unconditionally calling `get()` should be safe here.